pom.xml

```diff
@@ -497,6 +497,12 @@
         <groupId>org.glassfish.jersey.test-framework</groupId>
         <artifactId>jersey-test-framework-core</artifactId>
         <version>${jersey.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>jakarta.activation</groupId>
+            <artifactId>jakarta.activation-api</artifactId>
+          </exclusion>
```

Comment:
FYI:
@waitinfuture @pan3793 
I exclude `jakarta.activation-api` from the `test` scope dependency `jersey-test-framework-core`.

Then the `jakarta.activation-api` disappear from `dependencies-server`.

It does not make senses, right?

Reply:
but the finally dependencyList of maven and sbt are the same.

Reply:
I checked the existing dependencies involved in this pr, all of them appear in the kyuubi dependency list.

So, I think the dependencies looks fine.